lib/route/image/index.js

```diff
@@ -101,24 +101,25 @@ router.get("/:id.:ext", function (req, res, next) {
                             serveImage(imageEntry[0].data.buffer, false);
                         });
                 } else if (mimeType === "image/bmp") {
-                    const bitmap = bmp.decode(imageEntry[0].data.buffer, true);
-                    sharp(bitmap.data, {
-                        raw: {
-                            width: bitmap.width,
-                            height: bitmap.height,
-                            channels: 4,
-                        },
-                    })
-                        .toFormat("png")
-                        .png({ quality: 40 })
-                        .toBuffer()
-                        .then((imageBuffer) => {
-                            serveImage(imageBuffer, true);
+                    try {
+                        const bitmap = bmp.decode(imageEntry[0].data.buffer, true);
+                        sharp(bitmap.data, {
```

Comment:
I think I need to `await` this in order to have errors be caught by the try/catch handler. I think perhaps what I'll do is extract `bmp.decode` and `sharp` calls into a separate function, so that I don't need to make the entire callback function `async`.